ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -151,6 +160,55 @@ open class BottomCommandingController: UIViewController {
     /// The object that acts as the delegate of this controller.
     @objc open weak var delegate: BottomCommandingControllerDelegate?
 
+    /// Sets the `isHidden` property with a completion handler.
+    /// - Parameters:
+    ///   - isHidden: The new value.
+    ///   - animated: Indicates if the change should be animated. The default value is `true`.
+    ///   - completion: Closure to be called when the state change completes.
+    @objc public func setIsHidden(_ isHidden: Bool, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil) {
+        guard isViewLoaded else {
+            _isHidden = isHidden
+            return
```

Comment:
minor: this feels like an odd use of "guard" since it actually does some work. Though I'm definitely conservative when it comes to early returns

Reply:
I got rid of the guard.